client/src/features/action/assets/Asset.tsx

```diff
@@ -204,11 +207,11 @@ function Asset(): JSX.Element {
             {asset.title}
           </Typography>
           <Typography className={classes.subText} variant="subtitle1">
-            {asset.categories.join(', ')}
+            {/*{asset.categories.join(', ')} */}
           </Typography>
           <Typography className={classes.subText} variant="subtitle1">
             {asset.organization ? '' : 'Posted By:'}
-            {asset.postedBy.firstName}
+            {/*{asset.postedBy.firstName} */}
```

Comment:
in asset.entity change to `@ManyToOne(() => User, (user) => user.assets, { eager: true })`
and then you should get the `poster` property with a user object as value on client side
will have to change react Asset type to have `poster` not `postedBy`

Reply:
Changed and is working now.

Reply:
I changed to poster in Asset type and needed to change first name in User type to first_name as well. I also changed it in several inbox directory files.